server.js

```diff
@@ -21,12 +21,22 @@ app.listen(PORT, console.log("server is on"));
 const weatherData = require('/weather/weather.js');
 const { response } = require('express');
  */
+//const getLocationData = ('/location.json');
 
 //==========================================================================
 
-app.get('/data/location.js', getLocationData)
```

Comment:
This is where the problem you were talking about came from. nice lesson learned.